code/modules/mob/living/carbon/slime/slime.dm

```diff
@@ -535,6 +538,23 @@
 	var/datum/reagents/R = new/datum/reagents(100)
 	reagents = R
 	R.my_atom = src
+	icon_state_backup = icon_state
+	if (Uses > 1)
+		update_icon()
+
+
+/obj/item/slime_extract/update_icon()
+	..()
+	if (Uses == 1||Uses<0) //return if 1 or less uses
```

Comment:
not your fault, I hate that Uses is capitalized unlike everywhere else in the code where variables are generally all lower case